src/main/java/io/getunleash/repository/FeatureRepositoryImpl.java

```diff
@@ -201,4 +231,35 @@ public WasmResponse<VariantDef> getVariant(String toggleName, UnleashContext con
     public Stream<FeatureDefinition> listKnownToggles() {
         return this.engine.listKnownToggles().stream().map(FeatureDefinition::new);
     }
+
+    public synchronized void handleStreamingUpdate(String data) {
```

Comment:
decided to put this function here similar to updateFeatures for non streaming processing. The reson for that is that we have access to engine and dispatcher in current scope